### PR TITLE
Add pauseOnHover prop

### LINKED ISSUE #30: Enhancement: pauseOnHover prop needed
Hey, I really like this library, could you guys provide a prop to pause the notification so to read a notification with more content on hovering, I don't want to change the dismiss time

### PR DESCRIPTION
**Pull request by davidbucka**
closes #30 

Hi I don't know if this feature is wanted, but I just had some spare time and added it as I saw the feature request in the issues :)

### CODE REVIEW

File: src/ToastController.js [hunk 1]
```diff
@@ -50,6 +61,8 @@ export class ToastController extends Component<Props, State> {
       <Transition appear mountOnEnter unmountOnExit timeout={time} {...props}>
         {transitionState => (
           <Toast
+            onMouseOver={() => this.onMouseOver()}
```
Comment (jossmac) [inline src/ToastController.js]:
```suggestion
            onMouseOver={this.onMouseOver}
```

File: src/ToastController.js [hunk 2]
```diff
@@ -50,6 +61,8 @@ export class ToastController extends Component<Props, State> {
...
+            onMouseLeave={() => this.onMouseLeave()}
```
Comment (jossmac) [inline src/ToastController.js]:
```suggestion
            onMouseLeave={this.onMouseLeave}
```

File: src/ToastController.js (hunk 1 above)
Comment (davidbucka) [reply src/ToastController.js]: ahh yes that makes sense - I changed it just now 👍

File: src/ToastController.js (hunk 2 above)
Comment (davidbucka) [reply src/ToastController.js]: changed it too

File: src/ToastController.js (hunk 1 above)
Comment (jossmac) [reply src/ToastController.js]: Any reason you went with `onMouseOver` rather than `onMouseEnter`?

Comment (davidbucka) [reply src/ToastController.js]: Not really, but as I think about it I will change it just to be on the safe side regarding bubbling events

**Comment by jossmac** [comment]
Released in `v1.4.0`

**Pull request merged**